tests/mocks/feature_store_mock.py

```diff
@@ -90,9 +90,9 @@ def _batch_get_record_stub(self, *args, **kwargs):
                     errors.append({'ErrorMessage': str(e)})
 
         if errors:
-            return {'Records': [], 'Errors': errors}
+            return {'Records': records, 'Errors': errors}
         else:
-            return {'Records': []}
+            return {'Records': records}
```

Comment:
This was a bug in the mock feature store that went unnoticed because it was only used with `empty_feature_group.json`.